src/libs/ajax/AzureStorage.ts

```diff
@@ -19,7 +19,7 @@ type StorageDetails = {
   sas: SasInfo;
 };
 
-interface AzureFileRaw {
+export interface AzureFileRaw {
```

Comment:
this shouldn't need to be exported

Reply:
actually it did to silence that TS error/warning in my IDE (IntelliJ)... it was complaining in ajax.js root file that something being used there was depending on this secondary type.